.github/workflows/app_build.yml

```diff
@@ -272,12 +278,33 @@ jobs:
       # We'll use this as our working directory for all subsequent commands
       shell: bash
       run: |
+        set -x
         cmake -E make_directory ./build
         # Below needed if you re-enable 'Download the OpenStudio installer' step
         # openstudio_DIR=$(find `pwd`/OpenStudio-$OS_SDK_VERSION/ -name "openstudioConfig.cmake")
         # openstudio_DIR=$(dirname "${openstudio_DIR}")
         # echo "openstudio_DIR=${openstudio_DIR}"
         # echo "::set-env name=openstudio_DIR::$openstudio_DIR"
+        echo "Install aqtinstall, then install Qt 5.15.0 for ${{ matrix.QT_OS_NAME }} ${{ matrix.QT_ARCH }}"
+        pip3 install aqtinstall
+        aqt install --outputdir ./build/Qt-install/ 5.15.0 ${{ matrix.QT_OS_NAME }} desktop ${{ matrix.QT_ARCH }} -m qtwebengine qtwebglplugin
+        if [ "$RUNNER_OS" == "Windows" ]; then
+          echo "::add-path::$(pwd)/build/Qt-install/5.15.0/msvc2019_64"
```

Comment:
```suggestion
          echo "::add-path::$(pwd)/build/Qt-install/5.15.0/msvc2019_64/bin"
```

Reply:
Why is bin/ needed on windows?

Reply:
I didnt test on windows directly, but I find that weird. Cmake during the find_package calls will look into lib/cmake

Reply:
@macumber

Reply:
I downloaded Qt 5.15 on my computer using aqtinstall.  All of the tools like `moc` and `qmake` as well as the Qt dlls are under `bin`.  I assumed that is what we needed to add to the path?

Reply:
Yeah ok that makes sense. Actually I adding the path to root not thinking about binaries.

I thought about changing this to remove `NO_DEFAULT_PATHS` so it woul just pick up any installed (system) Qt if you do not supply QT_INSTALL_DIR explicitly: https://github.com/openstudiocoalition/OpenStudioApplication/blob/97ca38d046531ca27a7f058fdf275c1bb5767fb8/CMakeLists.txt#L475